src/index.html

```diff
@@ -17,6 +26,83 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="https://www.moyo.ua/ua/">
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav">
+          <li class="nav__item">
+            <a
+              href="https://www.moyo.ua/ua/portal_brand/apple_1/"
```

Comment:
You can fix href everywhere
```suggestion
              href="#apple"
```